HeavyIonsAnalysis/JetAnalysis/python/FullJetSequence_nominalPP.py

```diff
@@ -13,23 +13,21 @@
 from RecoJets.JetProducers.PFJetParameters_cfi import *
 from RecoJets.JetProducers.AnomalousCellParameters_cfi import *
 akSoftDrop4PFJets = cms.EDProducer(
-    "FastjetJetProducer",
+    "SoftDropJetProducer",
```

Comment:
Is SoftDropJetProducer simply the FastjetJetProducer with the useSoftDrop flag always on?